modules/sink/router/config/router.xml

```diff
@@ -15,12 +15,13 @@
 
 
 	<beans:beans profile="use-expression">
-		<router input-channel="input" expression="${expression}"/>
+		<router input-channel="input" expression="${expression}" resolution-required="false"
+						default-output-channel="nullChannel"/>
 	</beans:beans>
 
 	<beans:beans profile="use-script">
 		<beans:import resource="../../../common/script-variable-generator.xml"/>
-		<router input-channel="input">
+		<router input-channel="input" resolution-required="false" default-output-channel="nullChannel">
```

Comment:
I think it would be nice to have a configuration option to specify the default output channel based on whether the messages should be discarded via `nullChannel` or put into `errorChannel`. Once we have a stream specific errorChannel (the feature specified in https://jira.spring.io/browse/XD-2135) then, it would all make sense.